pyanalyze/test_annotations.py

```diff
@@ -428,5 +428,24 @@ def test_type(self):
 def capybara(x: Type[str], y: "Type[int]"):
     assert_is_value(x, SubclassValue(str))
     assert_is_value(y, SubclassValue(int))
+"""
+        )
+
+    @skip_before((3, 9))
+    def test_generic_alias(self):
+        self.assert_passes(
+            """
+from queue import Queue
+from typing import List
```

Comment:
I guess `List` was not needed after all?

Reply:
Yes, the error reproduced without it